ktor-shared/ktor-serialization/ktor-serialization-kotlinx/ktor-serialization-kotlinx-protobuf/common/test/ProtoBufSerializationTest.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+import io.ktor.http.*
+import io.ktor.http.content.*
+import io.ktor.serialization.*
+import io.ktor.serialization.kotlinx.*
+import io.ktor.serialization.kotlinx.protobuf.*
+import io.ktor.test.dispatcher.*
+import io.ktor.util.reflect.*
+import kotlinx.serialization.*
+import kotlinx.serialization.protobuf.*
+import org.junit.*
+
+@Serializable
+internal data class User(val id: Int, val login: String)
+
+@Serializable
+internal data class Photo(val id: Long, val path: String)
+
+@ExperimentalSerializationApi
+class ProtoBufSerializationTest {
+
+    @Test
+    fun testRegisterCustom() = testSuspend {
+        val serializer = KotlinxSerializationConverter(DefaultProtoBuf)
+
+        val user = User(1, "user1")
+        val actual = serializer.testSerialize(user)
+
+        Assert.assertArrayEquals(byteArrayOf(8, 1, 18, 5, 117, 115, 101, 114, 49), actual)
+    }
+
+    @Test
+    fun testRegisterCustomList() = testSuspend {
+        val serializer = KotlinxSerializationConverter(DefaultProtoBuf)
+
+        val user = User(2, "login2")
+        val photo = Photo(3, "photo3.jpg")
+
+        ProtoBuf.encodeToByteArray(listOf(user))
+        Assert.assertArrayEquals(
```

Comment:
please use static imports for assert functions

Reply:
done